wgpu-core/Cargo.toml

```diff
@@ -19,6 +19,7 @@ trace = ["ron", "serde", "wgt/trace", "arrayvec/serde", "naga/serialize"]
 replay = ["serde", "wgt/replay", "arrayvec/serde", "naga/deserialize"]
 # Enable serializable compute/render passes, and bundle encoders.
 serial-pass = ["serde", "wgt/serde", "arrayvec/serde"]
+webgl = ["hal/gles"]
```

Comment:
Why do we need this at all, can't we just assume gl is on wasm all the time as wgc isn't built on a webgpu build?